redbot/cogs/economy/economy.py

```diff
@@ -932,6 +934,7 @@ async def paydaytime(self, ctx: commands.Context, seconds: int):
 
         - `<seconds>` The new number of seconds to wait in between uses of payday. Default is 300.
```

Comment:
The example above and the description of the `<seconds>` parameter here should be updated to reflect the type of input that the used converter accepts.

Reply:
finished